lib/lutaml/model/key_value_mapping.rb

```diff
@@ -16,10 +16,17 @@ def map(
         render_default: false,
         with: {},
         delegate: nil,
-        child_mappings: nil
+        child_mappings: nil,
+        id: nil
       )
         validate!(name, to, with)
-
+        uniq_id = SecureRandom.hex(8)
```

Comment:
@0arm0 why do we need to use `SecureRandom`? This is an unnecessary dependency. I don't see why we can't number the names sequentially.